src/transformers/models/dpt/image_processing_dpt.py

```diff
@@ -275,10 +282,162 @@ def _get_pad(size, size_divisor):
 
         return pad(image, ((pad_size_left, pad_size_right), (pad_size_top, pad_size_bottom)), data_format=data_format)
 
+    def reduce_label(self, label: ImageInput) -> np.ndarray:
+        label = to_numpy_array(label)
+        # Avoid using underflow conversion
+        label[label == 0] = 255
+        label = label - 1
+        label[label == 254] = 255
+        return label
```

Comment:
This seems to be fully copied from beit image processor, you should add a `# Copied from` statement above if that's the case :)

Reply:
Done